src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerListImages.groovy

```diff
@@ -16,13 +16,12 @@
 package com.bmuschko.gradle.docker.tasks.image
 
 import com.bmuschko.gradle.docker.response.ResponseHandler
-import com.bmuschko.gradle.docker.response.image.ListImagesResponseHandler
 import com.bmuschko.gradle.docker.tasks.AbstractDockerRemoteApiTask
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Optional
 
 class DockerListImages extends AbstractDockerRemoteApiTask {
-    private ResponseHandler<Void, List<Object>> responseHandler = new ListImagesResponseHandler()
+    private ResponseHandler<Void, List<Object>> responseHandler
```

Comment:
Not really related to what you put together but I was thinking we just remove `ResponseHandler`. Ben had a use-case for this at one point but that never came to fruition. Don't think we really need it and this is a better solution IMO. Thoughts?